vocabularies/quality.ttl

```diff
@@ -0,0 +1,685 @@
+# baseURI: https://linked.data.qld.gov.au/vocab
+# imports: https://linked.data.qld.gov.au/def/dcat
+
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dq-iso-19157: <https://linked.data.gov.au/def/ISO-19157/quality-dimension/> .
+@prefix dq-iso-25012: <https://linked.data.gov.au/def/ISO-IEC-25012/quality-dimension/> .
+@prefix dq-nsw: <https://linked.data.gov.au/def/NSW-OEH/quality-dimension/> .
+@prefix dqv: <http://www.w3.org/ns/dqv#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix sdo: <http://www.schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+dq-iso-19157:Dimensions
+  a skos:ConceptScheme ;
+  dcterms:source "ISO 19157"@en ;
+  rdfs:label "Dimensions of data quailty (ISO 19157)"@en ;
+  skos:prefLabel "Dimensions of data quailty (ISO 19157)"@en ;
+  dcterms:created "2020-11-16"^^xsd:date ;
+  dcterms:creator <http://linked.data.gov.au/def/gsq-alias/des> ;
+dcterms:modified "2020-11-16"^^xsd:date ;
+dcterms:publisher <http://linked.data.gov.au/def/gsq-alias/des> ;

```

Comment:
```suggestion
  dcterms:created "2020-11-16"^^xsd:date ;
  dcterms:creator <http://linked.data.gov.au/org/des> ;
dcterms:modified "2020-11-16"^^xsd:date ;
dcterms:publisher <http://linked.data.gov.au/org/gsq> ;

```
I think the structure above is what you intended.
Question. Is the publisher DES via our system, or is the GSQ publishing DES content using our system?

Reply:
I'll go with your suggestion, I think this is what we meant to do.